packages/view/README.md

```diff
@@ -1,3 +1,36 @@
 # `@thorgate/spa-view`
 
 > View helpers used by [Thorgate project template SPA variant](https://gitlab.com/thorgate-public/django-project-template/tree/spa)
+
+Wrap your components into Views using withView helper to take advantage of:
+* ErrorBoundary (see [errors](../errors/README.md))
+* Managing and restoring scroll position so that contents does not jump on loading
```

Comment:
This only happens when it is not supported by the browser or has been turned off.